compose/um-metrics-02/docker-compose.yml

```diff
@@ -0,0 +1,110 @@
+version: "3.9"
+
+volumes:
+  prometheus-data: {}
+  grafana-data: {}
+  um_data: {}
+  um_logs: {}
+  um_licence: {}
+  um_users: {}
+
+networks:
+  n1:
+    external: false
+
+services:
+  umnode:
+    image: ${UM_CONTAINER_IMAGE_NAME}
+    ports:                         
+     - 9000:9000
+     - 9200:9200
+    volumes:
+     - um_data:/opt/softwareag/UniversalMessaging/server/umserver/data
+     - um_logs:/opt/softwareag/UniversalMessaging/server/umserver/logs
+     - ${UM_LICENCE_LOCATION}:/opt/softwareag/UniversalMessaging/server/umserver/licence
+     - um_users:/opt/softwareag/common/conf
+    networks:
+      n1:
+        aliases:
+          - "um-node"
+
+  prometheus:
+    image: prom/prometheus
+    ports:
+      - 9090:9090
+    volumes:
+      - ./prometheus/:/etc/prometheus/
+      - prometheus-data:/prometheus
+    command:
+      - '--config.file=/etc/prometheus/prometheus.yml'
+      - '--storage.tsdb.path=/prometheus'
+      - '--web.console.libraries=/usr/share/prometheus/console_libraries'
+      - '--web.console.templates=/usr/share/prometheus/consoles'
+    depends_on:
+      - cadvisor
+    networks:
+      n1:
+        aliases:
+          - "prometheus"
+    mem_reservation: 1g
+    mem_limit: 2g
+    cpus: 1
+  
+  # Server that produces metrics data on the host
+  node-exporter:
+    image: prom/node-exporter:latest
+    container_name: node-exporter
+    restart: unless-stopped
+    volumes:
+      - /proc:/host/proc:ro
+      - /sys:/host/sys:ro
+      - /:/rootfs:ro
+    command:
+      - '--path.procfs=/host/proc'
+      - '--path.rootfs=/rootfs'
+      - '--path.sysfs=/host/sys'
+      - '--collector.filesystem.mount-points-exclude=^/(sys|proc|dev|host|etc)($$|/)'
+    ports:
+      - 9100:9100
```

Comment:
```suggestion
      - ${H_SUIF_PORT_PREFIX}00:9100
```

Reply:
Actually the besides the Grafana port, and maybe UM, any other port may be not mapped at all...